gateway-api/api/nearby-spots.js

```diff
@@ -0,0 +1,31 @@
+//nearby-spots.js
+
+module.exports = function(db) {
+  return function(req, res) {
+    var superagent = require('superagent');
+
+    superagent
+    .get('http://localhost:3000/spots')
+    if(err) {
+      res.status(500).json({
+        code: 'DbError',
+        err: err
+      });
+    } else {
+        res.json(spots);
+        var spotsToReturn = [];
+        for(var i = 0; i < spots.length; i++) {
+          if (res.body.location[0] > spots[i].location[0][0] &&
```

Comment:
Add some comments like [0] is top left, and [0][0] is longitude, with [0][1] is latitude